### PR TITLE
add java 8 check to gradlew

### PR DESCRIPTION
**Pull request by sin3point14**
### Contains

Adds a check for Java 8 to gradlew

### How to test

Set your JAVA_HOME env var to point to some other Java version and watch gradlew complain about it

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: build.gradle [hunk 1]
```diff
@@ -23,6 +23,11 @@ buildscript {
     }
 }
 
+// Check for Java 8
+if(JavaVersion.current() != JavaVersion.VERSION_1_8){
```
Comment (skaldarnar) [inline build.gradle]: Please move this a couple of lines lower (to line 46) next to the compatibility check.

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
Looking good, I really like that it stands out in red! I did adjust the actual message a bit, we want to get away from _forcing_ use of Java 8 as soon as possible, which will also be the general idea in the future, so the wording is a bit softer now :-) Hope that's OK with you - thanks!

JavaWarning

**Pull request merged**